components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -1112,7 +1114,7 @@ subroutine ocn_diagnostic_solve_vorticity(dt, normalVelocity, tangentialVelocity
 
       nCells = nCellsAll
       nVertices = nVerticesAll
-      nEdges = nCellsAll
+      nEdges = nEdgesAll
```

Comment:
is this a bug fix?  The original definitely seems wrong, but that change is not really a part of the PR ti seems.

Reply:
Yes, this is a bug fix.